yellowbrick/features/pca.py

```diff
@@ -207,6 +214,10 @@ def finalize(self, **kwargs):
         self.ax.set_ylabel('Principal Component 2')
         if self.proj_dim == 3:
             self.ax.set_zlabel('Principal Component 3')
+        if self.legend_labels is not None:
```

Comment:
To keep consistent with the other changes, let's change this to a check to see if `self.classes_` exists, and if so, add a manual legend.